Database/Base/ShardBase.sql

```diff
@@ -313,251 +245,168 @@ CREATE TABLE `ace_position` (
+  KEY `idx_landblock_raw` (`landblockRaw`),
+  KEY `idx_landblock` (`landblock`),
+  KEY `idx_cell` (`cell`),
+  CONSTRAINT `fk_position_ao` FOREIGN KEY (`aceObjectId`) REFERENCES `ace_object` (`aceObjectId`) ON DELETE CASCADE
+) ENGINE=InnoDB AUTO_INCREMENT=43262 DEFAULT CHARSET=utf8;
 
---
--- Table structure for table `ace_weenie_class`
---
+/*Table structure for table `ace_weenie_class` */
 
 DROP TABLE IF EXISTS `ace_weenie_class`;
-/*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
+
 CREATE TABLE `ace_weenie_class` (
   `weenieClassId` int(10) unsigned NOT NULL,
   `weenieClassDescription` text NOT NULL,
   PRIMARY KEY (`weenieClassId`),
   UNIQUE KEY `idx_weenieName` (`weenieClassDescription`(100))
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-/*!40101 SET character_set_client = @saved_cs_client */;
 
---
--- Temporary view structure for view `vw_ace_object`
---
+/*Table structure for table `vw_ace_character` */
+
+DROP TABLE IF EXISTS `vw_ace_character`;
+
+/*!50001 DROP VIEW IF EXISTS `vw_ace_character` */;
+/*!50001 DROP TABLE IF EXISTS `vw_ace_character` */;
+
+/*!50001 CREATE TABLE  `vw_ace_character`(
+ `guid` int(10) unsigned ,
+ `accountId` int(10) unsigned ,
+ `NAME` text ,
+ `deleted` tinyint(1) ,
+ `deleteTime` bigint(20) unsigned ,
+ `weenieClassId` int(10) unsigned ,
+ `weenieClassDescription` text ,
+ `aceObjectDescriptionFlags` int(10) unsigned ,
+ `physicsDescriptionFlag` int(10) unsigned ,
+ `weenieHeaderFlags` int(10) unsigned ,
+ `itemType` int(10) unsigned 
+)*/;
+
+/*Table structure for table `vw_ace_object` */
 
 DROP TABLE IF EXISTS `vw_ace_object`;
-/*!50001 DROP VIEW IF EXISTS `vw_ace_object`*/;
-SET @saved_cs_client     = @@character_set_client;
-SET character_set_client = utf8;
-/*!50001 CREATE VIEW `vw_ace_object` AS SELECT 
- 1 AS `aceObjectId`,
- 1 AS `name`,
- 1 AS `weenieClassId`,
- 1 AS `weenieClassDescription`,
- 1 AS `aceObjectDescriptionFlags`,
- 1 AS `physicsDescriptionFlag`,
- 1 AS `weenieHeaderFlags`,
- 1 AS `itemType`,
- 1 AS `positionId`,
- 1 AS `positionType`,
- 1 AS `LandblockRaw`,
- 1 AS `posX`,
- 1 AS `posY`,
- 1 AS `posZ`,
- 1 AS `qW`,
- 1 AS `qX`,
- 1 AS `qY`,
- 1 AS `qZ`*/;
-SET character_set_client = @saved_cs_client;
-
---
--- Temporary view structure for view `vw_ace_weenie_class`
---
+
+/*!50001 DROP VIEW IF EXISTS `vw_ace_object` */;
+/*!50001 DROP TABLE IF EXISTS `vw_ace_object` */;
+
+/*!50001 CREATE TABLE  `vw_ace_object`(
+ `aceObjectId` int(10) unsigned ,
+ `name` text ,
+ `weenieClassId` int(10) unsigned ,
+ `weenieClassDescription` text ,
+ `aceObjectDescriptionFlags` int(10) unsigned ,
+ `physicsDescriptionFlag` int(10) unsigned ,
+ `weenieHeaderFlags` int(10) unsigned ,
+ `itemType` int(10) unsigned ,
+ `positionId` int(10) unsigned ,
+ `positionType` smallint(5) unsigned ,
+ `LandblockRaw` int(10) unsigned ,
+ `posX` float ,
+ `posY` float ,
+ `posZ` float ,
+ `qW` float ,
+ `qX` float ,
+ `qY` float ,
+ `qZ` float 
+)*/;
+
```

Comment:
is vw_ace_object still a thing?  can it be removed?  Same for many of the tables below.